userguide/tutorials/what_is_kill_bill.adoc

```diff
@@ -2,26 +2,30 @@
 
+While Kill Bill is used for various cases (b2b, b2c, offline, etc.), to get a better sense of what Kill Bill offers out of the box, the easiest is to look at how it is used in a typical b2c e-commerce environment.
 
-== Overview
+The *subscription billing* features allow an online store to create recurring or usage based subscriptions (e.g. "gold plan subscription at $9.99/month") and manage upgrades and downgrades. The system will take care of generating invoices, managing credits and triggering payments. Netflix, Dropbox and The New York Times are examples of websites which sell such subscriptions.
+
+The *payment* features let the store connect with virtually any payment gateway and processor (Stripe, Braintree, PayPal, Adyen, etc.) to charge your customers (using credit or debit cards, ACH, wire transfers, Bitcoin, etc.) for their subscriptions or one-off payments (e.g. shopping cart-type websites like Amazon or Zappos).
 
 Check this video for a quick tour:
 
 https://www.youtube.com/watch?v=f2IHcz3OLYo[image:http://img.youtube.com/vi/f2IHcz3OLYo/0.jpg[align=center]]
 
+== Overview
+
 === Big Picture
 
-At a high level we provide 2 pieces of software that **you will have to deploy and manage** in your own data center or in any public cloud.
-//TODO For convenience we also offer a managed version running in AWS that you can use for free to get started very quickly.
+At a high level, we provide 2 pieces of software that **you will have to deploy and manage** in your own data center, in your private cloud, or in any public cloud.
 
-* Kill Bill, or Kill Bill server is a standalone Java server that runs in your back-end. Your customers will never interact with it directly, but instead your website (it could be a custom e-commerce website, a Drupal or WordPress deployment, etc.) will trigger REST API calls (over HTTP) to process subscriptions or orders.
-* KAUI, our Back-office UI, which provides an interface to interact with Kill Bill for your support and finance teams (to manage refunds, invoice adjustments, etc.).
+* Kill Bill, or Kill Bill server, is a standalone Java server that runs in your back-end. Your customers will never interact with it directly, but instead your website (it could be a custom website, a Drupal or WordPress deployment, etc.) and potentially your internal tools (CRM, CPQ, etc.) will trigger REST API calls (over HTTP) to create subscriptions, retrieve invoices, give credits, etc.
```

Comment:
```suggestion
* Kill Bill, or Kill Bill server, is a standalone Java server that runs in your back-end environment. Your customers never interact with Kill Bill directly. Instead, by using your website (it could be a custom website, a Drupal or WordPress deployment, etc.) and potentially your internal tools (CRM, CPQ, etc.), they trigger REST API calls (over HTTP) to create subscriptions, retrieve invoices, make payments, etc.
```

Reply:
Reads funny to me? Are you sure about the _By using your website and your internal tools, they..._ ?

To be clear, there are two bits here: on one hand the website and on the other the internal tools. Customers will only interact through the website, company employees would be the ones using the internal tools.